CHANGES

```diff
@@ -1,11 +1,15 @@
 
 For a complete changelog, see:
 
-* https://github.com/yaml/pyyaml/commits/
+* https://github.com/Quansight-Labs/pyyaml/commits/
 
-7.0.0.dev0 (TBD)
+7.0.0 (2025-04-28)
 
-* TBD
+* https://github.com/Quansight-Labs/pyyaml/pull/2 -- Mark extension as free-threading compatible
+  and resolve Python-level threading issues
+* https://github.com/Quansight-Labs/pyyaml/pull/4 - Build cp313 and cp313t wheels
+* https://github.com/Quansight-Labs/pyyaml/pull/5 - Rename to PyYAML-ft everywhere
+* https://github.com/Quansight-Labs/pyyaml/pull/7 - Add CI job to publish to PyPI
```

Comment:
Is there other new stuff we should note about that we’re merged into the development branch since 6.0 or the last stable release?